### DISCUSSION TITLE
Documentation API

### DISCUSSION THREAD

**Discussion by monkey3d** [Koenkk/zigbee2mqtt#11979] [General]
Hi,
In Zigbee2MQTT 1.17.0 a new API has been introduced, documentation of the old API can be found here.
For information, the given link results in a 404 error.
Having developed my own home automation solution in php with the Symfony framework, I would like to decode the messages to be able to control different modules. This would complete the modules of other technologies that I have already integrated.
The current examples in the documentation don't seem to completely match the received messages. I am using version 1.24.0 and practicing on the "Livarno Lux smart LED light strip 2.5m" module.
Is there more detailed documentation indicating the format of the messages or even their sequencing?
Thank you.

**Comment by Koenkk** [comment]
> The current examples in the documentation don't seem to completely match the received messages. 

Could you give an example?

**Comment by monkey3d** [comment]
I think I read the documentation too quickly. I hadn't seen the explanations of the "options" and "exposes" properties of "definition" for, for example, the bridge/devices message.
It is on these points that I did not see the descriptions.
Sorry for my previous message which was useless.
I therefore continue the integration of zigbee modules in my home automation solution under php (Symfony) with the php mqtt client.
Great job on your project.
Thank you.
